odk1-src/form-language.rst

```diff
@@ -0,0 +1,128 @@
+.. spelling::
+
+  Anaranjado
+  Azul
+  Español
+  Púrpura
+  Qué
+  Rojo
+  Seleccione
+  colores
+  gustan
+  prefs
+  te
+  tres
+	
+Form Localization
+===================
+
+:doc:`collect-intro` and `XLSForm`_ support `multi-language forms`_.
+
+.. _multi-language forms: http://xlsform.org/#language
+
+To add additional languages to your XLSForm,
+replace columns of user-facing content with language-specific columns.
+
+All columns representing user-facing text or media can be multi-lingual:
+
+ - :th:`label`
+ - :th:`hint`
+ - :th:`media::*`
+ - :th:`constraint_message`
+ - :th:`required_message`
+
+Each language column adds two colons and the language name,
+followed by the `two letter language code` in parenthesis.
+For example: :th:`::English (en)`
```

Comment:
The colon after the `example` followed by two colons looks a little weird. Maybe add more text or another language so it's a wee bit clearer?

> For example, to make a column French, add <b>:: French (fr)</b>

OR

> For example: <b>:: English (en)</b> or <b>:: French (fr)</b>